run/events-pubsub/Program.cs

```diff
@@ -16,25 +16,29 @@
 using Microsoft.AspNetCore.Hosting;
 using Microsoft.Extensions.Hosting;
 
-namespace HelloEvents
+namespace EventsPubSub

```

Comment:
nit: We have started moving away from having namespaces on samples, just because it eases readability and copy/pasting.

Reply:
So remove all namespaces?

Reply:
Ok, removed the namespace.